src/tools/reloading.tsx

```diff
@@ -133,32 +145,28 @@ export function reloading(WrappedComponent) {
       const counter = this.state.listMode ? 1 : this.state.counter
       const perPage = this.state.listMode ? this.state.maxCount : 1
       const nextCounter = counter < this.state.maxCount ? counter + 1 : 1
-      const url = resolveToApiUrl(
-        this.props.match,
-        counter,
-        perPage,
-        this.state.since,
-        this.state.hashtags
-      )
+      const url = resolveToApiUrl(this.props.match, counter, perPage, this.state.since, this.state.hashtags)
 
       this.setState({ counter: nextCounter }, () => {
         // If demo data is requested do not query the API
         if (this.state.demo) {
           console.log(`Demo Mode: API Request would have been: "${url}".`)
           return this.storeData(demoData(this.props.match, counter, perPage))
         }
-        fetch(url)
-          .then(response => response.json())
-          .then(json => this.storeData(json))
-          .then(undefined, err => console.log(err))
+        if (url) {
+          fetch(url)
+            .then((response) => response.json())
+            .then((json) => this.storeData(json))
+            .then(undefined, (err) => console.log(err))
+        }
```

Comment:
With above guard clause we can revert this.
```suggestion
        fetch(url)
          .then((response) => response.json())
          .then((json) => this.storeData(json))
          .then(undefined, (err) => console.log(err))
```